app/scripts/controllers/transactions/index.js

```diff
@@ -1003,7 +1003,7 @@ export default class TransactionController extends EventEmitter {
 
       if (submittedTime) {
         metricsParams.completion_time = this._getTransactionCompletionTime(
-          submittedTime,
+          txMeta.time,
```

Comment:
`txMeta.time` is the time the transaction is created

The issue here is that we need to show the actual time the transaction is confirmed

Reply:
unless... does `txMeta.time` get updated anywhere?

Reply:
no, I don't think so

Reply:
nope, txMeta.time is the timestamp from etherscan when TX  starts mining.